### ISSUE TITLE
Add COFF executable support

### ISSUE THREAD

**Issue by TheServer201** [x64dbg/x64dbg#822]
Oda can do it https://www.onlinedisassembler.com/odaweb/ so why not x6'dbg.

**Comment by mrexodia** [comment]
Notice that x64dbg is only for dynamic analysis so I don't quite see the point 😀

**Comment by TheServer201** [comment]
Yes but for example I actually use sse instruct and gcc or clang do shit xD so I need to only see a function (in a coff) so it can be usefull to add this feature.

**Comment by mrexodia** [comment]
But how exactly is this related to debugging? X64dbg doesn't have any kind
of static analysis and I don't really want to add any.
